config/checkstyle-input-suppressions.xml

```diff
@@ -19,38 +19,16 @@
   <suppress id="noCheckstyleInInputs"
             files="[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityExcludedPackagesDirectPackages.java"/>
   <!-- until https://github.com/checkstyle/checkstyle/issues/4845 -->
-  <suppress id="noCheckstyleInInputs"
-          files="[\\/]checks[\\/]imports[\\/]customimportorder[\\/]InputCustomImportOrderThirdPartyAndSpecial.java"/>
   <suppress id="noCheckstyleInInputs"
           files="[\\/]checks[\\/]imports[\\/]customimportorder[\\/]InputCustomImportOrderThirdPartyAndSpecial2.java"/>
```

Comment:
all Inputs that we are not going to update as they need same package by design, please move above `<!-- until https://github.com/checkstyle/checkstyle/issues/4845 -->`, to know that all of them are due to `<!-- permanent suppression as import needs to be of same package where class is -->`

Reply:
sure

Reply:
done @romani